core-tests/shared/src/test/scala/zio/ChunkSpec.scala

```diff
@@ -57,6 +57,53 @@ object ChunkSpec extends ZIOBaseSpec {
         assert(chunk.size)(equalTo(chunk.length))
       }
     ),
+    suite("add")(
```

Comment:
We should probably test `Chunk.empty + 2` (given that empty is an edge case in Chunk).

Reply:
The `chunkOf` generator generates a good number of empty chunks so I think we are getting quite a bit of coverage for this but can add another test.

Reply:
Ah, no need then! Just was at the top of my mind.